src/config.rs

```diff
@@ -0,0 +1,17 @@
+/// These commands are executed when no arguments are given to `cargo watch`
+pub const DEFAULT_COMMANDS: [&'static str; 2] = ["build", "test"];
```

Comment:
The default commands don't make sense, after thought. `cargo test` _implies_ `cargo build`, so we may as well just run `cargo test` alone by default.